eos-tech-support/eos-install-google-chrome

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash -e
+
+# Downloads Google Chrome and unzips to the user's home directory,
+# with a desktop launcher
+
+exit_with_error() {
+   echo $1
+   exit 1
+}
+
+USERID=$(id -u)
+if [ "$USERID" == "0" ]; then
+    exit_with_error "Program cannot be run as superuser"
+fi
+
+DEB_ARCH=`dpkg --print-architecture`
+case ${DEB_ARCH} in
+    amd64) ;; # Only 64-bit Intel architectures are supported
+    *) exit_with_error "Unsupported architecture ${DEB_ARCH}"
+       ;;
+esac
+
+TEMP_DIR=`mktemp -d`
+DEST_DIR=~/.google-chrome
+WGET_COMMAND="wget -c --tries=5"
+
+CHROME_URL=http://dl.google.com/linux/chrome/deb
+CHROME_PACKAGES=${CHROME_URL}/dists/stable/main/binary-${DEB_ARCH}/Packages
+CHROME_FLAVOUR_VERSION=google-chrome-stable
+CHROME_VERSION=
+CHROME_SHA1SUM=
+CHROME_URL_FILENAME=
+CHROME_FILENAME=
+
+GCONF_URL=http://httpredir.debian.org/debian
+GCONF_PACKAGES=${GCONF_URL}/dists/stable/main/binary-${DEB_ARCH}/Packages.xz
+GCONF_FLAVOUR_VERSION=libgconf-2-4
+GCONF_VERSION=
+GCONF_SHA1SUM=
+GCONF_URL_FILENAME=
+GCONF_FILENAME=
+
+get_field() {
+    grep-dctrl -n -s $1 -P $2 ${TEMP_DIR}/Packages
+}
+
+download_packages_file() {
+    local packages_url=$1
+    local packages_filename=$(basename $1)
+    if ! ${WGET_COMMAND} "${packages_url}" -O "${TEMP_DIR}/${packages_filename}"; then
+        exit_with_error "Failed to download Packages file ${packages_filename} "
+    fi
+    if [[ ${packages_filename} == *.xz ]]; then
```

Comment:
Because the Packages file from Chrome's repository is a plain text `Packages` file, not a compressed `Packages.xz`. This check just makes sure we always end up with a plain text `Packages` file in both cases